src/ssz/type/container.zig

```diff
@@ -74,6 +76,17 @@ pub fn FixedContainerType(comptime ST: type) type {
             return true;
         }
 
+        /// Creates a new `FixedContainerType` and clones all underlying fields in the container.
+        ///
+        /// Caller owns the memory.
+        pub fn clone(
+            _: std.mem.Allocator,
```

Comment:
can you remove the (unused) allocator param from here and other fixed types? (made a few suggestions but you probably get the idea, we're using the same pattern for deserialization, fromJson, etc)

Reply:
addressed in [88f2fb0](https://github.com/ChainSafe/ssz-z/pull/51/commits/88f2fb0366abfeaaeb296bc736982d700f89feed)